pkg/process/metadata/parser/service.go

```diff
@@ -69,7 +71,7 @@ type ServiceExtractor struct {
 
 type serviceMetadata struct {
 	cmdline        []string
-	serviceContext string
+	serviceContext []string
```

Comment:
I can separate but it in this case I've to stack this on top of that one. I did everything in one for simplicity and because this suggestion arose during the review of that PR

Reply:
That's a breaking change for the algorithm, I think we should move the code to a separate PR
especially as we have concerns with the multiple tags in general (the thread in slack)